e2e/node-logger-e2e/tests/logger-client.spec.ts

```diff
@@ -0,0 +1,240 @@
+            ).not.toThrow();
+        }, 120000);
+
+        it('should create src with http transport being set to true and httpTransportHost being set', async () => {
+            const project = uniq('node-logger');
+            await runNxCommandAsync(
+                `generate @ensono-stacks/node-logger:node-logger ${project} --httpTransport --httpTransportHost=localhost`,
+            );
+            expect(() =>
+                checkFilesExist(
+                    `libs/${project}/src/index.ts`,
+                    `libs/${project}/src/index.test.ts`
+                    ),
+            ).not.toThrow();
+        }, 120000);
+
+        it('should create src with http transport being set to true and httpTransportPort being set', async () => {
+            const project = uniq('node-logger');
+            await runNxCommandAsync(
+                `generate @ensono-stacks/node-logger:node-logger ${project} --httpTransport --httpTransportPort=3000`,
+            );
+            expect(() =>
+                checkFilesExist(
+                    `libs/${project}/src/index.ts`,
+                    `libs/${project}/src/index.test.ts`
+                    ),
+            ).not.toThrow();
+        }, 120000);
+
+        it('should create src with http transport being set to true and httpTransportPath being set', async () => {
+            const project = uniq('node-logger');
+            await runNxCommandAsync(
+                `generate @ensono-stacks/node-logger:node-logger ${project} --httpTransport --httpTransportPath=somePath`,
+            );
+            expect(() =>
+                checkFilesExist(
+                    `libs/${project}/src/index.ts`,
+                    `libs/${project}/src/index.test.ts`
+                    ),
+            ).not.toThrow();
+        }, 120000);
+
+        it('should create src with http transport being set to true and httpTransportSSL being set to true', async () => {
+            const project = uniq('node-logger');
+            await runNxCommandAsync(
+                `generate @ensono-stacks/node-logger:node-logger ${project} --httpTransport --httpTransportSSL`,
+            );
+            expect(() =>
+                checkFilesExist(
+                    `libs/${project}/src/index.ts`,
+                    `libs/${project}/src/index.test.ts`
+                    ),
+            ).not.toThrow();
+        }, 120000);
+    });
+
+    describe('--streamPath', () => {
+        it('should create src with stream path being set', async () => {
+            const project = uniq('node-logger');
+            await runNxCommandAsync(
+                `generate @ensono-stacks/node-logger:node-logger ${project} --streamPath=/somePath`,
+            );
+            expect(() =>
+                checkFilesExist(
+                    `libs/${project}/src/index.ts`,
+                    `libs/${project}/src/index.test.ts`
+                    ),
+            ).not.toThrow();
+        }, 120000);
+    });
+});
```

Comment:
For all these test cases where it's testing each passed flag one by one, I wonder if we could optimize it and test all flags in one go? You're already testing them properly in unit tests, so I don't think there's a need to repeat that here and it could significantly cut down the time to run it:

![Screenshot 2023-02-06 at 17 52 16](https://user-images.githubusercontent.com/802141/217033808-f6580c3a-c80c-4998-a135-faa7043a8704.png)

Each of the `--httpTransport` tests took 7s to run :o Curious why, as the others are quite quick, but nevertheless, I think maybe one single command should cover it all?